### ISSUE TITLE
support structure plugin

### ISSUE THREAD

**Issue by gokul2907** [Ultimaker/Cura#2402]
Is there any plugin available to calculate the volume of support structure generated

**Comment by fieldOfView** [comment]
No, there is not. Do you want the volume of the extruded material to print the support, or the volume of the entire support structure?

**Comment by gokul2907** [comment]
Is it possible to get the volume of the extruded material to print the support structure .

**Comment by fieldOfView** [comment]
If you print the support structure with a secondary extruder, the amount of material is shown in the interface (length and grams & price if you have entered the appropriate data for the materials).

**Comment by Ghostkeeper** [comment]
Yes, but then the support might also be printed with different settings so you have to be careful about that.

A Cura plug-in for the amount of material in the support is fairly difficult to make because it involves parsing the g-code that we receive. A feature could be built in CuraEngine though that sends over the material usage per feature, so we won't have to calculate it in the front-end.

**Comment by Ghostkeeper** [comment]
Our project manager refused this enhancement because it was not considered important enough.
